files/zh-tw/web/demos/index.md

```diff
@@ -54,8 +54,8 @@ Mozilla 支援豐富、廣泛且扣人心弦的網路公開技術，並且我們
 
 ### 虛擬實境 ( Virtual Reality )
 
-- [The Polar Sea](http://mozvr.com/demos/polarsea/) ([source code](https://github.com/MozVR/polarsea))
-- Sechelt fly-through ([source code](https://github.com/mozvr/sechelt))
+- The Polar Sea ([source code](https://github.com/MozVR/polarsea))
+- [Sechelt fly-through](http://mozvr.github.io/sechelt/) ([source code](https://github.com/mozvr/sechelt))
```

Comment:
This doesn't seem to match the [upstream changes](https://github.com/mdn/content/pull/31117/files#diff-ea6d28c20bdbced2cf87451ebbb3c642144e1e5deab95559121030122eddd4cbR44).

Reply:
Yes, but http://mozvr.github.io/sechelt/ is working link, I found it in `mdn/translated-content`))
So I think it is useful to keep it.  And it would be good to make PR to `mdn/content`. And also add in all locales(+ fix `http` to `https`).

Reply:
PR to `en-us`: https://github.com/mdn/content/pull/31123